pkg/server/token_minter_test.go

```diff
@@ -330,6 +354,49 @@ func TestValidateRepositories(t *testing.T) {
 	}
 }
 
+func TestAllowRequestAllRepos(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name    string
+		allow   []string
+		request []string
+		want    bool
+	}{
+		{
+			name:    "allow all repos",
+			allow:   []string{"*"},
+			request: []string{"*"},
+			want:    true,
+		},
+		{
+			name:    "disallow all repos",
+			allow:   []string{"test1"},
+			request: []string{"*"},
+			want:    false,
+		},
+		{
+			name:    "allow all only with * request",
+			allow:   []string{"*"},
+			request: []string{"test1"},
+			want:    false,
+		},
+	}
+
+	for _, tc := range cases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+
+			got := allowRequestAllRepos(tc.allow, tc.request)
+			if got != tc.want {
+				t.Errorf("expected %t to be %t", got, tc.want)
```

Comment:
nit: I tend to find the following easier to read
```suggestion
				t.Errorf("allowRequestAllRepos got=%t, want=%t", got, tc.want)
```